core/modules/entity/entity.module

```diff
@@ -349,18 +349,20 @@ function entity_extract_ids($entity_type, $entity) {
  *
  * @return
  *   An entity object, initialized with the IDs provided.
+ *
+ * @deprecated since 1.15.0.
  */
 function entity_create_stub_entity($entity_type, $ids) {
-  $entity = new stdClass();
```

Comment:
Could you add a deprecated watchdog call in here as the first line?
```
watchdog_deprecated_function('entity', __FUNCTION__);
```

Reply:
Done